server.js

```diff
@@ -84,6 +84,17 @@ app.use(
     referrerPolicy: false,
   })
 )
+
+// Helper to generate the redirects to the right document in the new docs paths
+hbs.registerHelper('new-docs', (currentPage) => {
+  // This particular page is the root for the docs in the new site
+  if (!currentPage || currentPage.endsWith('tutorial/introduction')) {
+    return '/docs/latest/'
+  } else {
+    return currentPage.replace('docs/', 'docs/latest/')
```

Comment:
Are there any docs/latest/ links anywhere that will get adjusted here to docs/latest/latest/ ?

Reply:
The /docs/latest/ route is handled by the new site and no request will end up in the express server so `currentPage` can never be `docs/latest/*` when reaching this code :)